swagger_server/controllers/cryptography/keys.py

```diff
@@ -86,3 +86,6 @@
 -----END ENCRYPTED PRIVATE KEY-----
 
 """
+
+
+TEST_KEYS_YML="""{"keys": {"2225f01580a949801274a5f3e6861947018aff5b": {"platform": "sdc", "type": "public", "purpose": "inbound", "value": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAu4UO1frTr8LPyyKXkQNi\nua4VrCeLec/sMkkYt7aFf/jMr2pTCNeyGGD0lp1JZp5u4fs3yVMoQTxfmswggYG/\n44pzNYK7+03GyvGbNrHcLXZ1nPETPhMPWobWOWlQ0YH9vLCLD4nuFUS7ma5qTiRX\ny/UKPPvs5MSYj5EOFqkqziToTzsQ9LJewsLODSvQuILcX9Ygb0seq9zxesEoOPgz\n0xhxht8kTIp24UMhax1QAwOtqjkiShUOnj8xnXjGoc1GOJSSLDGm9dxaOt1F4uC5\nddIEdYRuxd9A7yWjP1I8jWC/DSq8UEAooVfe6XNsoWWF2zn9XDjhWw0hwEXSX/QT\n9wIDAQAB\n-----END PUBLIC KEY-----\n", "service": "sdx", "use": "encryption", "version": "v1"}, "fe425f951a0917d7acdd49230b23a5c405c28510": {"platform": "sdc", "type": "private", "purpose": "inbound", "value": "-----BEGIN RSA PRIVATE KEY-----\nMIIEowIBAAKCAQEAq0rFub/fC2S4kcP4l9ivxgljh8l3hYOKG5z8mJ1cggoRqTaI\npeS3d8eM4zLpuY0Pp5GMulHdDTjfoguPOSB2OJiYUm8QFwiq2L4C92TsteRXaatJ\nWqORyH2KP8X7q09fqtuIiBXvD9kwA9Miid+Z9Q/Ye950GTaJEJ0norqZpZEcIRdm\nlZIelqXJxE8OV+HbEojccLU7RaL7avCeVnK3jhzQmofAmEF3rLqEWHpctyuO0TfE\nEw8Obxs2+klsQWnsfq3LhB7grm3dbTrTWBOScPolFkuI7TbabIFcOxy0gxJHmF9T\n1aPaym/g0KVOLhfAEI52DV7q4cywHv4FCK6dBwIDAQABAoIBAB4/fP0ZMdT8O0+a\nUKrX2+pXvBi9bfrIyLmplj2JBE3+YzIDbEwHQJLJTUrNsdJPY6VOoZBBZoywL2WZ\nLviCj/3cjIF6FVhL8oeFvlZbnyVbpkgFtFVqLxM9gA1U8+VDhfxQ+iI15VpiGwWB\n4kaAktpgqvTsyomtRLUHW3l0BC/lTvf2nQw+TgjCp5DGy2VZPrzDonBIxppD12Du\n2a4FYukz2Pnz7vCFHpEepz+NWmmmfRHMs/LgxCCEk/LfE3ySUmNdagCB784BJxJM\nNus1vrgurJE9MWOt1TQ8lyBNtpaGo5pTk0VFcvFwe5lNp+fiMXvi1nLOGqUyssLC\nUxc1DYkCgYEA39n+FYNr86Xc8CcVUAQyr3MKDnk5TOTy6oUUXpJq0EZDoqguqRfg\n9iMlZyj/Qqzcot7AixwuBNFgALiEXspd3Q9uSLxuwLKsWeOrIsvUGZQwldItSID3\n7uwb8Qr3TOAZmOPipaQl71qcA4I08lq+Sp9Kfd0ENEW2WALTTC30fQUCgYEAw+Ro\nHAd6rxCIyDIh6nnk69PS9+iuR6H/6Bc0vU1aagszDQktpk0UDkjaaMQ6RHebjZjW\nszBrsD5EFT5GLxLj0vbUJvWzK6+6lqOzisdiIu0U11B3eL6NEvaAdsgSq9Qj4X6N\nrYh5x0W5E2VIoDAe1NiglJagiaHUvF82S15XL5sCgYEAkxI0P4oy+Md2m0nyVLoN\nWvxoFQ5MOVsKxk5dDa2BBAnGEi4RSgNoSA7bWiWzX+kWpONY+YywJ3YwbwjL30Nx\n2/VxbdRvp6c8WWqdW6UzWwLj0Otf2Qzy2ETELuAXjqiF6M+Ab3ejfNi5PAzmIHzC\nHWD+nicBxh4vKS/iB2ECCskCgYALtH40Do5aH5bMv5IfPKFdJg8ZGSIwzZFJAPn5\nxcF9TKYERWQg58InCFtknzUMGkmFMHjDxETrBw1yZ4IyFeLfHDiZC11lYt63WZjE\n1h0bESeJphk9ccVkOK58aoATHF34f2Nf1yPPsG7apdatG7L77OXLeySklLLtfpLz\n9fMnuQKBgFJClh2vNYFlNndXoV5NFPLNz1L5Wv5WNwWcuGO4iCsUh2jFVl43z459\nV7ZuRp//4vIzEONqqyLUFt1JZKuCwWQB2sRjKWpqRB07C0o243XefD5f7844nmXq\nH0xKMFRYjDqhFdTXFWVPnk/tu/MxoSTdK0FfsgUnjs+TGvN5TnBW\n-----END RSA PRIVATE KEY-----\n", "service": "ras", "use": "signing", "version": "v1"}}}"""
```

Comment:
Are these keys just for test purposes? I'm unclear if they should be under the test/ folder as a fixture file.